scripts/fetch_external_css_and_js.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# cd ..
+# URLs for e.g. blueprints are relative assuming you are in the scout folder (repo root)
+
+# CSS
+mkdir scout/server/blueprints/public/static/external
+# create a url list for later reference
+for url in git grep \\.css |grep "src\|href" |grep -v "url_for" |perl -ne 'm/href=\"([^"]+)/; print $1,"\n";'|sort|uniq; do echo $url >> tmp.css_urls; done
```

Comment:
🤔 
![image](https://user-images.githubusercontent.com/28093618/146750189-c940ca08-16b4-4262-94eb-0e62fbf102cb.png)

Reply:
Heh, some backticks were lost in translation.. I also apparently managed to merge the ldap support branch into this PR. While that is all good, lets take things in order.